core/src/main/java/amino/run/kernel/client/KernelClient.java

```diff
@@ -36,8 +36,9 @@
     /* TODO: Need to check if size of data need to be increased to get the better data transfer rates. Sometimes, empty
     heartbeats are taking longer time than heartbeat with 2k arbitrary data. Thus, resulting in negative data transfer
     rate */
-    private static final int RANDOM_DATA_SIZE = 2 * 1024;
-    private static final byte[] randomBytes = new byte[RANDOM_DATA_SIZE];
+    private static final int DATA_SIZE_IN_MB = 10; // size in MegaBytes
```

Comment:
You still need to explain here why 10MB has been chosen.

Reply:
Have modified it to start with 1k of data and increase by step size when the data transfer time is not significant when compared to latency.